tests/components/airzone/test_climate.py

```diff
@@ -179,6 +191,43 @@ async def test_airzone_climate_set_hvac_mode(hass):
         assert state.state == HVAC_MODE_OFF
 
 
+async def test_airzone_climate_set_hvac_slave_error(hass):
+    """Test setting the HVAC mode for a slave zone."""
+
+    HVAC_MOCK = {
+        API_DATA: [
+            {
+                API_SYSTEM_ID: 1,
+                API_ZONE_ID: 5,
+                API_ON: 1,
+            }
+        ]
+    }
+
+    await async_init_integration(hass)
+
+    with patch(
+        "homeassistant.components.airzone.AirzoneLocalApi.http_request",
+        return_value=HVAC_MOCK,
+    ):
+        try:
```

Comment:
use `with pytest.raises(HomeAssistantError)` instead

Reply:
Changed in 5421593f1e8fd25795cc93f64017acd8abdd7682